### DISCUSSION TITLE
Welcome to Alkemio Discussions!

### DISCUSSION THREAD

**Discussion by techsmyth** [alkem-io/alkemio#246] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FloriSpoelstra** [comment, marked as answer]
Very happy to be the first to introduce myself :-) I am participating here on behalf of Digicampus, launching partner of Alkemio. Very excited that we are now working on the platform in production, after all the hard work that has been done behind the scenes. Alkemio can be a huge catalyst for everyone working on innovation. The Digicampus-agenda revolves around the innovation of public (government) services in the Netherlands. The landscape is very fragmented and up till now there was no platform that could assist us in aligning activities and finding capabilities. Looking forward to further develop the platform with the community! 

Cheers,
Flori

Ps. Can we somehow pin this thread so it stays on top? Would be nice for new members I think.
